lib-injection/requirements.json

```diff
@@ -29,5 +29,1281 @@
       }
     ]
   },
-  "deny": []
-}
+  "deny": [
+    {
+      "id": "gem",
+      "description": "Ignore the rubygems CLI",
+      "os": null,
+      "cmds": [
+        "**/ruby"
+      ],
+      "args": [
+        {
+          "args": [
+            "**/gem"
```

Comment:
FYI, the `args` only use simple wildcard, not glob patterns, so this is functionally equivalent to `"*/gem"`

Reply:
Huh I could have sworn swear I changed that to use `*`.

Note: that inconsistency is confusing AF.